tests/tensorflow/test_models/densenet.py

```diff
@@ -11,10 +11,11 @@
  limitations under the License.
 """
 
-from tensorflow.python.keras import backend
-from tensorflow.python.keras import layers
-from tensorflow.python.keras.applications import imagenet_utils
-from tensorflow.python.keras.engine import training
+import tensorflow as tf
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+
+from tests.tensorflow.helpers import imagenet_utils
```

Comment:
Why don't you import it directly from `tf_internals`?

Reply:
That's right. These functions also need to be import differently by version and import in tf_interals.